File: decompiled/sources/com/hjq/http/request/PutRequest.java
package com.hjq.http.request;

import androidx.lifecycle.LifecycleOwner;
import com.hjq.http.model.HttpMethod;

public final class PutRequest extends BodyRequest<PutRequest> {
    public PutRequest(LifecycleOwner lifecycleOwner) {
        super(lifecycleOwner);
    }

    /* access modifiers changed from: protected */
    public String getRequestMethod() {
        return HttpMethod.PUT.toString();
    }
}

File: decompiled/sources/com/hjq/http/request/PutRequest.kt
package com.hjq.http.request

import androidx.lifecycle.LifecycleOwner
import com.hjq.http.model.HttpMethod

class PutRequest(
    lifecycleOwner: LifecycleOwner
) : BodyRequest<PutRequest>(lifecycleOwner) {
    override fun getRequestMethod(): String {
        return HttpMethod.PUT.toString()
    }
}
